include/bitcoin/explorer/commands/qrcode.hpp

```diff
@@ -125,7 +127,8 @@ class BCX_API qrcode
     virtual arguments_metadata& load_arguments()
     {
         return get_argument_metadata()
-            .add("PAYMENT_ADDRESS", 1);
+            .add("PAYMENT_ADDRESS", 1)
+            .add("filename", 1);
```

Comment:
Yes, prefer streaming output. The old bash script was from SX, which was sort of inconsistent in this regard. Remember the code you are writing is actually a library, not a console. The console wraps the library. The library is much more useful if it streams the output. It also improves testing and console flexibility.